app/code/Magento/UrlRewriteGraphQl/Model/Resolver/EntityUrl.php

```diff
@@ -73,6 +74,11 @@ public function resolve(
         $url = $customUrl ?: $url;
         $urlRewrite = $this->findCanonicalUrl($url);
         if ($urlRewrite) {
+            if (!$urlRewrite->getEntityId()) {
```

Comment:
Please, cover this changes with API-functional tests (`testGetNonExistentUrlRewrite`)

Reply:
@naydav sure, I will.

Reply:
Hi @naydav, I've created `testGetNonExistentUrlRewrite` test case to cover `custom` type URL rewrite rule. Please take a look.